connector/src/main/java/org/geysermc/connector/network/translators/ItemStackTranslator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-2020 GeyserMC. http://geysermc.org
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.connector.network.translators;
+
+import com.github.steveice10.mc.protocol.data.game.entity.metadata.ItemStack;
+import com.nukkitx.protocol.bedrock.data.ItemData;
+import org.geysermc.connector.network.session.GeyserSession;
+
+public class ItemStackTranslator {
+
+    public ItemStack translateToBedrock(GeyserSession session, ItemStack itemStack) {
+        return itemStack;
+    }
+
+    public ItemData translateToJava(GeyserSession session, ItemData itemData){
```

Comment:
Could this be refactored to directly return an `ItemStack` class instead of `ItemData`? Also from the looks of it, it doesn't seem like the GeyserSession passed in this constructor is used at all. Is this something that will be used in the future and if so, what?

Reply:
So you can always edit things before the tags are translated. You can also combine several tags by applying them in a sequence.